lib/licensed/sources/dep.rb

```diff
@@ -13,11 +13,6 @@ def enumerate_dependencies
           package_dir = @config.pwd.join("vendor", package[:name])
           search_root = @config.pwd.join("vendor", package[:project])
 
-          unless package_dir.exist?
-            next if @config.ignored?("type" => Dep.type, "name" => package[:name])
```

Comment:
`Licensed::Sources::Source#dependencies` already rejects dependencies that are ignored, and `Licensed::Dependency#initialize` now sets errors for packages that don't exist 🎉